svg/pservers/reftests/hatch-basic-001-ref.svg

```diff
@@ -0,0 +1,65 @@
+<svg id="svg-root"
+  width="100%" height="100%" viewBox="0 0 480 360"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:html="http://www.w3.org/1999/xhtml">
+  <g id="testmeta">
+    <title>Hatch: Basic hatch.</title>
+    <html:link rel="author"
+          title="Tavmjong Bah"
+          href="http://tavmjong.free.fr"/>
+  </g>
+
+  <style id="test-font" type="text/css">
```

Comment:
Please remove the unused style declarations (AKA, the `<style>` blocks) from all the files (references and tests), to keep them as simple as possible.